### PR TITLE
Add new profile header

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- using the `pay` flag visit your profile
- you should see your username and a blue circle in the header.

**Comment by huwshimi** [comment]
[Add profile header](https://trello.com/c/4qTx1Zzw/96-add-profile-header)

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8319/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8320/

File: jujugui/static/gui/src/app/components/expanding-row/expanding-row.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ class ExpandingRow extends React.Component {
     this.observer.observe(this.refs.inner, {childList: true, subtree: true});
   }
 
-  componentWillUpdate(nextProps, nextState) {
+  componentWillReceiveProps(nextProps, nextState) {
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/expanding-row/expanding-row.js]: :) Thanks!

**Comment by hatched** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**